tests/backups/test_backup.py

```
@@ -45,3 +45,30 @@ async def test_consolidate_conflict_varied_encryption(
+    (mount_dir := coresys.config.path_mounts / "backup_test").mkdir()
+    enc_tar = Path(copy(get_fixture_path("test_consolidate.tar"), tmp_path))
+    enc_backup = Backup(coresys, enc_tar, "test", None)
+    await enc_backup.load()
+
+    unc_tar = Path(copy(get_fixture_path("test_consolidate_unc.tar"), mount_dir))
+    unc_backup = Backup(coresys, unc_tar, "test", "backup_test")
+    await unc_backup.load()
+
+    enc_backup.consolidate(unc_backup)
+    assert (
+        f"Backup in backup_test and None both have slug d9c48f8b but are not the same!"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)